cloud_automation/java/dataflow/src/main/java/org/datacommons/dataflow/CloudBigtableOptions.java

```diff
@@ -0,0 +1,26 @@
+package org.datacommons.dataflow;
+
+import org.apache.beam.runners.dataflow.options.DataflowPipelineOptions;
+import org.apache.beam.sdk.options.Description;
+import org.apache.beam.sdk.options.ValueProvider;
+
+public interface CloudBigtableOptions extends DataflowPipelineOptions {
+  @Description("The Google Cloud project ID for the Cloud Bigtable instance.")
+  ValueProvider<String> getBigtableProjectId();
+
+  @SuppressWarnings("unused")
```

Comment:
Do we need these unused methods in the future?

Reply:
Yes they are used by the templating framework to set value of parameter